src/App.js

```diff
@@ -2,24 +2,30 @@ import React from 'react';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    date: new Date(),
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  componentDidMount() {
+    setInterval(() => {
+      // eslint-disable-next-line
```

Comment:
why did you disable eslint?

Reply:
it was in not edited code, so I didn't remove it